docs/source/usersguide/output/statepoint.rst

```diff
@@ -182,9 +176,9 @@ if (run_mode == MODE_EIGENVALUE)
 
 *do i = 1, n_tallies*
 
-    **/tallies/tally i/estimator** (*int*)
+    **/tallies/tally i/estimator** (*char[]*)
```

Comment:
Ditto, the same comment as before for meshes applies here to tally IDs. I haven't yet made it to the new code in this PR for the statepoint, but was under the impression you had switched to using user-specified IDs in the statepoint file in place of the internally-generated indices.